models/repo.go

```diff
@@ -199,6 +199,7 @@ type Repository struct {
 	Size          int64              `xorm:"NOT NULL DEFAULT 0"`
 	IndexerStatus *RepoIndexerStatus `xorm:"-"`
 	IsFsckEnabled bool               `xorm:"NOT NULL DEFAULT true"`
+	Topics        []string           `xorm:"TEXT JSON"`
```

Comment:
Why is this saved to database?

Reply:
For better performance when we list all topics for every repository on repositories list UI.